src/lib.rs

```diff
@@ -1,5 +1,8 @@
 #![no_std]
 #![cfg_attr(target_arch = "xtensa", feature(asm_experimental_arch))]
+#![feature(panic_info_message)]
```

Comment:
this will make the lib nightly only? If we select this feature based on the `print-defmt` feature we can avoid that when not needed

Reply:
Fixed. Thank you!